scripts/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -e
+
+basepath="${PWD}"
+artifacts="${basepath}/public"
+
+if [ -d ${artifacts} ]; then
+  rm -R ${artifacts}
+fi
+
+echo "Artifacts: ${artifacts}"
+echo "Branch:    ${GITHUB_REF}"
+echo "Build:     ${GITHUB_RUN_NUMBER}"
+echo "Region:    ${AWS_REGION}"
+echo "Bucket:    ${AWS_BUCKET}"
+
+mkdir -p ${artifacts}
+
+node --version
+node indexer/indexer.js
```

Comment:
This should go in after https://github.com/betaflight/firmware-presets/pull/21, and then use `yarn run index`, in order to ensure a consistent environment.

Reply:
Either or - it's an easy change once one or the other is in.

Reply:
@mikeller I'd like to merge this, and then I will raise another PR once the #21 is merged.